src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/EETypeNode.cs

```diff
@@ -103,14 +103,11 @@ public EETypeNode(NodeFactory factory, TypeDesc type)
             factory.TypeSystemContext.EnsureLoadableType(type);
         }
 
-<<<<<<< HEAD
         public static bool SupportsWritableData(TargetDetails target) => true;
 
         public static bool SupportsFrozenRuntimeTypeInstances(TargetDetails target)
             => SupportsWritableData(target);
 
```

Comment:
```suggestion
```
The writeable data stuff should take upstream versions.

Reply:
Done.